### PR TITLE
GPG-756: Add subheading to DB Integrity checks page

### PR DESCRIPTION
**Pull request by xonde**
Before
Before

After
After

### CODE REVIEW

File: GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml [hunk 1]
```diff
@@ -30,6 +30,8 @@
 <span class="govuk-caption-xl">Administration</span>
 <h1 class="govuk-heading-xl">Database integrity checks</h1>
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Do we still want to keep this header?

**Review by malinaslevoaca1 — APPROVED** [review]
Nice! 👍

File: GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml (hunk 1 above)
Comment (xonde) [reply GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Good question, I'm not sure, Jacob only said he'd like us to add subheadings is this something we'd want to check?

Comment (malinaslevoaca1) [reply GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Yes, can you please check with Jacob?

Comment (xonde) [reply GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Should I reach him via slack or via the tickets comment section?

Comment (xonde) [reply GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Jacob's happy with the Heading currently, so all good

Comment (malinaslevoaca1) [reply GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml:31]: Great, thanks for checking!

**Pull request merged**